packages/grid/src/vaadin-grid-styles.js

```diff
@@ -189,6 +189,32 @@ export const gridStyles = css`
     overflow: hidden;
   }
 
+  /* Empty state */
+
+  #scroller:not([show-empty-state]) #emptystatebody,
+  #scroller[show-empty-state] #items {
+    display: none;
+  }
+
+  #emptystatebody {
+    display: flex;
+    position: sticky;
+    inset: 0;
```

Comment:
I'm not sure why the `position` and `inset` properties are needed here. Could you please explain?

Reply:
Ah, I see, thanks.